src/pages/nft/collection/_collectionId/index.vue

```diff
@@ -287,20 +298,29 @@ export default {
       if (hasStock) {
         try {
           this.isOpeningCheckoutPage = true;
+          const customPriceInDecimal = this.customPrice
+            ? this.formatCustomPrice(this.customPrice, this.collectionPrice)
+            : undefined;
+          const totalPrice =
+            (customPriceInDecimal || this.collectionPrice) * this.quantity;
           const purchaseEventParams = {
             items: [
               {
                 name: this.collectionName,
-                price: this.collectionPrice,
+                price: customPriceInDecimal || this.collectionPrice,
                 collection: this.collectionId,
               },
             ],
-            price: this.collectionPrice,
+            price: totalPrice,
             currency: 'USD',
             isNFTBook: true,
           };
```

Comment:
Is this replaced by `this.purchaseEventParams`?

Reply:
Yes